packages/build/tests/install/tests.js

```diff
@@ -153,15 +153,8 @@ test('Install local plugin dependencies: missing plugin in netlify.toml', async
   t.snapshot(normalizeOutput(output))
 })
 
-test('In integration dev mode, install local plugins and install the integration when forcing build', async (t) => {
-  const output = await new Fixture('./fixtures/local_missing_integration')
-    .withFlags({
-      context: 'dev',
-      testOpts: {
-        cwd: './tests/install/fixtures/local_missing_integration/',
-      },
-    })
-    .runWithBuild()
+test.only('In integration dev mode, install local plugins and install the integration when forcing build', async (t) => {
```

Comment:
😱 @ndhoule quick fix and maybe find the missing eslint rule to enable? 😬

Reply:
Oof, sorry about this! I'll fix it this morning.